app/src/main/java/org/wikipedia/page/leadimages/PageHeaderView.java

```diff
@@ -71,6 +75,15 @@ public void setCallback(@Nullable Callback callback) {
         this.callback = callback;
     }
 
+    public void setUpCallToAction(String callToActionText) {
+        if (callToActionText == null) {
+            callToActionContainer.setVisibility(GONE);
+        } else {
+            callToActionContainer.setVisibility(VISIBLE);
+            callToActionTextView.setText(callToActionText);
```

Comment:
Has this been resolved?

Reply:
This is not changed because, when we have the situation where :

you open an image which is not in commons [ add caption not shown]
move to the next image which happens to be in commons [ add caption shown]
now you move back, we need a way to hide it , or else the button is still showing when the image is not editable.